setup.py

```diff
@@ -43,9 +43,8 @@
     use_scm_version={"write_to": os.path.join("pyirf", "_version.py")},
     packages=find_packages(exclude=['pyirf._dev_version']),
     install_requires=[
-        "astropy>=4.0.2",
-        "matplotlib",
-        "numpy>=1.18",
+        "astropy>=5.3,<7.0.0a0",
```

Comment:
Superseded by #274, remainder of this PR is still valid though and should work with astropy 5.3.

Reply:
Updated